canvas_api_client/canvas_api_client/interface.py

```diff
@@ -123,6 +124,7 @@ def associate_courses_to_blueprint(self,
     @abstractmethod
     def get_account_blueprint_courses(self,
                                       account_id: str,
+                                      is_sis_course_id: bool = False,
```

Comment:
why this addition? just curious, i mean i'm not using this anymore but i'm worried this could break something. iirc i don't think this parameter is needed here..

Reply:
I would argue that if we're putting this method out there, we should either support it by keeping it up to date, or remove it.

Reply:
i'm ok adding it as long as we have backwards compatibility. it's not entirely clear to me that we have that yet.

Reply:
Please check my unit tests for that!

Reply:
I left your unit test, which should assure backwards compatibility.  I just added a new unit test for the sis_account_id.

Reply:
looks good to me!